src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -111,14 +112,83 @@
             <form:checkbox path="showNotesToStudents" id="showNotesToStudents"/> Show Notes entered on Class Roster page to students
         </label>
         <br/>
+            <label>
+                <input type ="checkbox" id="conversionConfirm"/> Convert Attendance to Assignment
+            </label>
+        <br/>
+
+        <div class = "container-fluid" id="conversionConfig" style="display: none">
+            <br/>
+            <label> NOTE: When this assignment is pushed to the gradebook, it will immediately be published. Please do not alter the assignment in the gradebook, but instead use this application to update the assignment as needed.
+            </label>
+            <br/>
+            <div class="col-md-2 col-md-offset-0">
+                <label for="assignmentName">
+                    <h5><i>Assignment Name: </i></h5>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)